app/src/main/res/values-it/strings.xml

```diff
@@ -171,216 +171,205 @@ Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
 
     <!-- Bluetooth Setting: Text unten -->
-    <string name="bluetooth_setting_data_explanation_text2">"Wichtige Eigenschaften:
-- Kein Location-Tracking
-- Keine Analytics oder Advertising Dienste
-- Offene Architektur / Protokoll und Spezifikation inkl. SDK für iOS und Android
-- Interoperabilität (Eigene App, eigenes Backend)
-- Open Source"</string>
-    <string name="reports_title_homescreen">"Meldungen"</string>
+    <string name="bluetooth_setting_data_explanation_text2">Caratteristiche importanti :\n- Nessun tracciamento di posizione\n- Nessun tracciamento di marketing ne pubblicità\n- Architettura aperta, protocollo, specifiche e SDK pubblicati\n- Completa interoperabilità\n- Open Source</string>
+    <string name="reports_title_homescreen">Report</string>
 
     <!-- Keine Meldungen Titel -->
-    <string name="meldungen_no_meldungen_title">"Keine Meldungen"</string>
+    <string name="meldungen_no_meldungen_title">Nessun report</string>
 
     <!-- Keine Meldungen Text -->
-    <string name="meldungen_no_meldungen_text">"Du wirst beim Verdacht auf eine mögliche Ansteckung informiert."</string>
+    <string name="meldungen_no_meldungen_text">Sarai informato in caso di possibile infezione.</string>
 
     <!-- Push deaktiviert Titel -->
-    <string name="push_deactivated_title">"Push deaktiviert"</string>
+    <string name="push_deactivated_title">Notifiche push disattivate</string>
 
     <!-- Push deaktiviert Text -->
-    <string name="push_deactivated_text">"Erlaube Push Nachrichten, um bei einer Meldung direkt informiert zu werden."</string>
+    <string name="push_deactivated_text">Abilita le notifiche push per essere informato immediatamente quando un report viene ricevuto.</string>
 
     <!-- Hinweis Titel -->
-    <string name="hinweis_title">"Wichtiger Hinweis"</string>
+    <string name="hinweis_title">Note</string>
 
     <!-- Hinweis Text -->
-    <string name="hinweis_text">"Eine Person, der du begegnet bist, wurde positiv auf COVID-19 getestet."</string>
-    <string name="button_report_illness">"Erkrankung melden"</string>
-    <string name="meldungen_meldung_title">"Wichtiger Hinweis"</string>
-    <string name="meldungen_meldung_text">"Eine Person, der du begegnet bist, wurde positiv auf COVID-19 getestet. Bitte informiere dich hier über die Verhaltensempfehlungen."</string>
-    <string name="meldungen_meldung_more_button">"Mehr erfahren"</string>
-    <string name="tracing_active_title">"Tracing aktiv"</string>
-    <string name="tracing_active_text">"Begegnungen werden anonym auf deinem Gerät gespeichert."</string>
-    <string name="tracing_error_title">"Tracing nicht aktiv"</string>
-    <string name="tracing_error_text">"Aktiviere Tracing und Bluetooth, damit Begegnungen anonym aufgezeichnet werden können."</string>
+    <string name="hinweis_text">Una persona con cui hai avuto contatto è stata testata positiva al COVID-19</string>
```

Comment:
Do you think would be better to change "è stata testata" with "è risultata"?

Reply:
si certo.